src/cpp/desktop/DesktopWebView.cpp

```diff
@@ -132,6 +148,14 @@ qreal WebView::dpiAwareZoomFactor()
 
 bool WebView::event(QEvent* event)
 {
+   if (event->type() == QEvent::ShortcutOverride)
+   {
+      // take a first crack at shortcuts
+      QKeyEvent *pKeyEvent = dynamic_cast<QKeyEvent*>(event);
```

Comment:
I think you can safely use `static_cast` rather than `dynamic_cast` here.

Reply:
You're right, this is probably overly pessimistic (Qt uses `static_cast` in their [own example](https://wiki.qt.io/ShortcutOverride)). I'll make the change.